src/Pages/ProductDetail/Component/ImgPurchInfo.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import './ImgPurchInfo.scss';
+
+class ImgPurchInfo extends Component {
+    constructor(){
+        super();
+        this.state={
+            price: 22000,
+            quantity: 0,
+            sizeBtn: true,
+        }
+    }
+
+    handleClick = () =>{
+        this.setState({
+            sizeBtn: !this.state.sizeBtn,
+        })
+    }
+
+    doNothing = () => {
+        return;
+    }
+
+    render() {
+        const {sizeBtn, price, quantity} = this.state
+
+        return (
+            <div className="ImgPurchInfo">
+                <div className="imgEx">
+                    <div className="bigImgContainer">
+                        <img className="bigImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
+                    </div>
+                    <div className="smallImgContainer">
+                        <img className="smallImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
+                        <img className="smallImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
+                        <img className="smallImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
+                        <img className="smallImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
+                        <img className="smallImg" alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
```

Comment:
* 똑같은 엘레먼트의 반복을 아래와 같이 바꿀수 있습니다. 
* alt도 생략하지 말고 어떤 이미지를 담았는지 알려주는 키워드로 설정해주세요 

```suggestion
{[...Array(5)].map((e, i) => <img key={i} className="smallImg" alt="" src="https://encryptedtbn0.gstatic.com/imagesq=tbn:ANd9GcSAdBUkL6CL9geDn8RcmxCtrjLzSIfL3seMFA&usqp=CAU"/>
            ))}
```